<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMsgServerClass</class>
 <widget class="QMainWindow" name="SendMsgServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SendMsgServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="ReceiveWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ReceiveMsg:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textReceive"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ReplyWidget" native="true">
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,2">
          <item row="0" column="2">
           <widget class="QTextEdit" name="textRplySet"/>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QTextEdit" name="textReply"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Reply Msg: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Reply Set: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkReplySet">
            <property name="text">
             <string>Same with receive,otherwise will disconnet after reply</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="BottomWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnOption">
            <property name="text">
             <string>Option</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>2888</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStart">
            <property name="text">
             <string>Start Server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStop">
            <property name="text">
             <string>Stop Server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear">
            <property name="text">
             <string>Clear Page</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionStart_Server">
   <property name="text">
    <string>Start Server</string>
   </property>
  </action>
  <action name="actionStopServer">
   <property name="text">
    <string>StopServer</string>
   </property>
  </action>
  <action name="actionChange_Reply_Message">
   <property name="text">
    <string>Change Reply Message</string>
   </property>
  </action>
  <action name="actionClear_Receive_Message">
   <property name="text">
    <string>Clear Receive Message</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sendmsgserver.qrc"/>
 </resources>
 <connections/>
</ui>
